src/include/uapi/ipc/control.h

```diff
@@ -152,4 +152,21 @@ struct sof_ipc_ctrl_data {
 	};
 } __attribute__((packed));
 
+/* event type */
+enum sof_ipc_ctrl_event_type {
+	SOF_CTRL_EVENT_KD_NOTIFICATION = 0,	/**< keyword detection event */
```

Comment:
should we use "Metadata" as well, so that we can reuse this for opaque values, e.g. when the kernel receives a "Metadata" it just passes the binary data as is.

Reply:
acknowledged